Daytime-Nighttime-Interactions/slack-channel-usage.rst

```diff
@@ -0,0 +1,196 @@
+.. For information that directly affects the current system users
+.. E.g. Resetting LOVE, starting to work with telescopes etc.
+.. Software updates, IT outages, use of generator power
+.. Required information regarding personnel or hardware safety
+.. Inclement weather, (?)
+.. System control handoffs
+.. Observatory State transitions (coming in the future)
+
+
+
+#summit-control-room
+--------------------
+This room is actively monitored by observing specialists and shift managers.
+
+Communications in this room should be minimal.
+This is a single channel for both telescopes which is actively monitored by telescope specialists and anyone controlling the system.
+For maximum efficiency, primary communication should be via Zoom and not in the channel.
+Breakout rooms can be used if multiple simultaneous conversations are required.
+
+This channel is meant to be used as the area where the copying/pasting of code snippets happens while supporting via zoom, therefore context will often not be provided.
+Troubleshooting in this area should be either system-wide, or issues which benefit from having multiple people see and understand the diagnosis.
+
+An example of troubleshooting belonging to this channel is found in `this Slack snippet <_static/dialog_example.png>`_.
+
+
+#summit-maintel and #summit-auxtel
+----------------------------------
+
+These rooms are for telescope system specific asynchronous conversation with remote support.
+They are passively monitored by Observing Specialists and there is to be no expectation of an immediate response.
+Generally, the content will be read and start and end of a shift, or during quiet periods.
+
+These channels are actively monitored by shift manager(s), who will directly contact with Observing Specialists if required.
+
+Example content:
+
+- Discussion of faults that were already recovered.
+  For example, "Experienced issue with shutter again, reported on OBS-12345."
+
+- Announcements such as, “Calibrations completed in filters XYZ” for people may be interested to know but do not follow #summit-announce. 
+
+- Announcements regarding future status of the system.
+  Such as, “TMA expected to be offline Wednesday night.”
+
+#auxtel-operations and #maintel-operations
+------------------------------------------
+
+These channels are also telescope system specific but are dedicated for communication between people providing external support, such as looking at data as it comes in.
+The external supporters are not directly tied in to on-site activities and are often not aware of the immediate status of the observatory.
+
+These channels are passively monitored by the shift managers, who elevate concerns/issues when required.
+Observing Specialists will often be participating as well, but are not required.
+On-shift Observing Specialists should not be @-'d at any time. 
+If required, contact the shift manager and they will take the issue to the real-time communication channel(s) if and when appropriate.
```

Comment:
I think you should consider dropping this part, it's just too inviting and we already say above the shift manager will elevate issues.

Reply:
oops, forgot to address this one, sorry.
My qualm here is that the channel rules will be linked from each channel individually (to these headings). So we need to repeat information at some level as I don't think people will actually read the entire thing (sadly).
On another note, we'll also need to link the schedule of shift managers... or people will just @channel and annoy everyone.